pages/src/landing/promo-banner/promo-banner.less

```
@@ -18,9 +18,16 @@
     opacity: .6;
 
     background: url('/assets/common/aurora.webp'), url('/assets/common/aurora.webp'), linear-gradient(rgba(0, 0, 0, 0), rgba(13, 57, 90, 1) 40%, rgba(0, 0, 0, 0));
-    background-position: right 157% bottom 22%,  left 150% bottom 18%, center;
-    background-size: 50vw 100vh, 50vw 85vh, auto;
+    background-position: right 157% bottom 22%, left 139% bottom 18%, center;
+    background-size: 50% 100%, 50% 85%, auto;
     background-repeat: no-repeat, no-repeat, repeat;
+    @media @xs {
+      background-position: right 162% bottom 22%, left 156% bottom 18%, center
+    }
+    @media @lg-xl and @xl {
```

```suggestion
    @media @lg-xl {
```

Just in case currently it's:
```suggestion
    @media @xl {
```